nullaway/src/main/java/com/uber/nullaway/fixserialization/XMLUtil.java

```diff
@@ -94,14 +94,25 @@ public static <T> DefaultXMLValueProvider<T> getValueFromTag(
     return new DefaultXMLValueProvider<>(null, klass);
   }
 
+  public DocumentBuilderFactory safeDocumentBuilderFactory() {
```

Comment:
This method should be `static`.  Also can we add a bit of Javadoc as to its purpose?